### PR TITLE
BETA: Register marioannier.is-a.dev

### PR DESCRIPTION
**Pull request by marioannier**
Added `marioannier.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/marioannier.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "marioannier",
+        "email": "mario.annier@gmail.com"
+    },
+    "record": {
+        "CNAME": "marioannier.github.io"
```
Comment (DEV-DIBSTER) [inline domains/marioannier.json:7]: Redirects back to this domain which you are requesting which results in loop...

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/marioannier.json (hunk 1 above)
Comment (marioannier) [reply domains/marioannier.json:7]: I just followed the steps:
Create the domain using the record type:
CNAME record 
```
{
  "record": {
    "CNAME": "<github-username>.github.io"
  }
}
```
After that I just configured like this: 

_To fix this, go to your GitHub pages repository's Settings > GitHub pages > Custom domain and add <your-subdomain>.is-a.dev in the given field. Only do this after your pull request is merged._
from: [HERE](https://www.is-a.dev/docs/)

Comment (marioannier) [reply domains/marioannier.json:7]: I understand now, why is a loop. I eliminate the .github.io -> is-a.dev redirection. Does this fix it ?

Comment (DEV-DIBSTER) [reply domains/marioannier.json:7]: > I understand now why is a loop. I eliminated the .github.io -> is-a.dev redirection. Does this fix it ?

The problem is that there is going to be no content displayed. When we assign a CNAME record to your domain, marioannier.is-a.dev would display the **content** on the CNAME record. Right now your CNAME record shows a 404 page, which in turn would show a 404 page when this record is created.

Comment (marioannier) [reply domains/marioannier.json:7]: I fixed that; my CNAME shows my personal web portfolio [LINK](https://marioannier.github.io/)

**Review by DEV-DIBSTER — APPROVED** [review]

**Pull request merged**